scripts/artifacts-building/containers/test-built-container.yml

```diff
@@ -0,0 +1,30 @@
+---
+- name: Prepare the testing env
+  hosts: localhost
+  connection: local
+  gather_facts: no
+  vars_files:
+    - container-vars.yml
+  vars:
+    image_name: "{{ lookup('env', 'CONTAINER_TYPE') | default('keystone', True) }}"
+    image_path: "{{ lxc_image_folder }}/{{ lxc_index_path }}/{{ image_name }}-{{ rpc_release }}/{{ build_id }}/"
+  tasks:
+    - name: Ensure test directory is created
+      file:
+        path: "/tmp/{{ image_name }}"
+        state: directory
+    - name: Copy rootfs into test directory
```

Comment:
Fair point, I was afraid of changes in the "built" rootfs (mtime, or anything that the unarchive module would do).
Because I didn't review unarchive module, I took the safe path. I could have a look.